src/state/contexts/GlobalState.tsx

```diff
@@ -89,15 +89,15 @@ export function GlobalStateProvider({
   const [updatingTicker, setUpdatingTicker] = useState(false);
 
   useEffect(() => {
-    if (state.ioTicker === initialState.ioTicker && !updatingTicker) {
+    if (!updatingTicker) {
```

Comment:
Not sure why there is updatingTicker at all here? What is it protecting against?

Reply:
[040afc3](https://github.com/ar-io/arns-react/pull/615/commits/040afc3e6a79e21822c6ca030ab167a64001f4b4)